### DISCUSSION TITLE
MQTT documentation

### DISCUSSION THREAD

**Discussion by brianlay714** [mikebrady/shairport-sync#1900] [General]
Is there any documentation of the MQTT interface besides the comments in the config file?  do the commands need to be formatted as lower case with quotes, such as  "play"?

**Comment by mikebrady** [comment]
Yes, there is an MQTT.md document…

**Comment by brianlay714** [comment]
Thanks Mike, I see that now.  I was looking for further details on the control capabilities using the remote topic.  Do commands need to be strictly formatted as lower case with quotes such as "play"? From looking at a discussion on the Home Assistant forum, it appears that is the case.
In my case, I'm using OpenHab and am sending commands to the remote topic.
Here's a screenshot from MQTT Explorer.
![image](https://github.com/user-attachments/assets/e713ac2c-8713-4712-8c39-0c3ab583a692)

**Comment by brianlay714** [comment]
Problem solved.
Converting to lower case did the trick!
|input.toLowerCase
